src/Behat/Behat/Output/Node/Printer/Html/templates/footer.html

```diff
@@ -0,0 +1,186 @@
+</div><!-- /suite -->
+
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.1/jquery.min.js"></script>
+<script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.1/js/bootstrap.min.js"></script>
+
+
+<script>
+
+    $(document).ready(function () {
+
+        /*
+
+         highlight v5
+
+         Highlights arbitrary terms.
+
+         <http://johannburkard.de/blog/programming/javascript/highlight-javascript-text-higlighting-jquery-plugin.html>
+
+         MIT license.
+
+         Johann Burkard
+         <http://johannburkard.de>
+         <mailto:jb@eaio.com>
+
+         */
+
+        jQuery.fn.highlight = function (pat) {
+            function innerHighlight(node, pat) {
+                var skip = 0;
+                if (node.nodeType == 3) {
+                    var pos = node.data.toUpperCase().indexOf(pat);
+                    pos -= (node.data.substr(0, pos).toUpperCase().length - node.data.substr(0, pos).length);
+                    if (pos >= 0) {
+                        var spannode = document.createElement('span');
+                        spannode.className = 'highlight';
+                        var middlebit = node.splitText(pos);
+                        var endbit = middlebit.splitText(pat.length);
+                        var middleclone = middlebit.cloneNode(true);
+                        spannode.appendChild(middleclone);
+                        middlebit.parentNode.replaceChild(spannode, middlebit);
+                        skip = 1;
+                    }
+                }
+                else if (node.nodeType == 1 && node.childNodes && !/(script|style)/i.test(node.tagName)) {
+                    for (var i = 0; i < node.childNodes.length; ++i) {
+                        i += innerHighlight(node.childNodes[i], pat);
+                    }
+                }
+                return skip;
+            }
+
+            return this.length && pat && pat.length ? this.each(function () {
+                innerHighlight(this, pat.toUpperCase());
+            }) : this;
+        };
+
+        jQuery.fn.removeHighlight = function () {
```

Comment:
My bad, they are not in use at the moment anyway so i've removed them until i can find a solid solution